src/components/Navbar.jsx

```diff
@@ -1,7 +1,9 @@
 import React, { useState } from 'react';
 
+import useDarkMode from "../hooks/useDarkMode"
+
 const Navbar = () => {
-  const [darkMode, setDarkMode] = useState(false);
+  const [darkMode, setDarkMode] = useDarkMode(false);
   const toggleMode = e => {
     e.preventDefault();
     setDarkMode(!darkMode);
```

Comment:
this then sets the value to be the opposite of what it currently is
!false = true 
!true = false